pink/configuration.py

```diff
@@ -103,6 +135,25 @@ def update(self, q: Optional[np.ndarray] = None) -> None:
             q_readonly = q.copy()
             q_readonly.setflags(write=False)
             self.q = q_readonly
+
+        # Compute collisions, if needed
+        if self.update_collision:
```

Comment:
We can do an equivalent check without the additional `update_collision` attribute:

```suggestion
        if self.collision_model is not None:
```

Unless we want users to see them (e.g. if it were convenient for users to set ``configuration.update_collision = False`` at runtime, but here I guess that would be a bad idea), it is better to have less public attributes in objects.